reactive/vpe_router.py

```diff
@@ -25,7 +25,7 @@
 @hook('config-changed')
 def validate_config():
     try:
-        if ('pass', 'vpe-router', 'user') not in cfg:
+        if not cfg.keys() & {'pass', 'vpe-router', 'user'}:
```

Comment:
This syntax is odd to me. maybe `if ['pass', 'vpe-router', 'user'] not in cfg.keys()` instead